problem_builder/mentoring.py

```diff
@@ -1040,6 +1018,16 @@ def update_num_attempts(self, data, suffix=''):
         }
 
     @XBlock.json_handler
+    def publish_attempt(self, data, suffix):
+        score = self.score
```

Comment:
@itsjeyd no tests as far as I can tell (hint: patch runtime.publish and verify it is called with expected values values when this method is called).